tests/e2e/test_api.py

```diff
@@ -604,6 +604,21 @@ def test_metrics() -> None:
     assert 'response_duration_seconds_sum{path="/metrics"}' in response.text
 
 
+@pytest.mark.cluster
+@pytest.mark.standalone
```

Comment:
the default is to run tests that don't have tags, so you really only need to tag it if we need to be able to exclude it.

(the cluster tests are tagged as such because they require auth to be functional, so they can't be run against standalone where auth doesn't exist)

Reply:
well, "Default" is the wrong way to describe it, but the expressions we use to pick the tests are all not-based.

so we are excluding tests w/ particular tags, not including tests w/ particular tags.

in any case, since this test can run anywhere, it does not need tags.

Reply:
actually it's not needed as we - as you wrote - can exclude tests, not include them. Initially I have in mind more specific approach based on `provider_*` marks, but it is not fully possible. Removed marks